cmd/sealer/cmd/build.go

```diff
@@ -60,48 +57,35 @@ build with args:
 	sealer build -f Kubefile -t my-kubernetes:1.19.8 --build-arg MY_ARG=abc,PASSWORD=Sealer123 .
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		buildContext := args[0]
+		builder, err := image_adaptor.NewAdaptor()
+		if err != nil {
+			logrus.Fatalf("failed to initiate a builder, %v", err)
+		}
 
-		targetPlatforms, err := platform.GetPlatform(buildConfig.Platform)
+		err = builder.Build(&buildFlags, args)
 		if err != nil {
+			logrus.Error(err)
```

Comment:
Error without wrap is unnecessary to log here, cobra framework will handle and print it.